src/python/pants/backend/python/dependency_inference/dependency_parser.py

```diff
@@ -14,7 +19,7 @@
 from pants.engine.rules import Get, MultiGet, collect_rules, rule
 from pants.util.logging import LogLevel
 
-# NOTE: Must call .format(min_dots=X) on this string to use it.
+# NOTE: Must call .format(import_min_dots=X, resource_min_slashes=Y) on this string to use it.
 _SCRIPT_FORMAT = """\
```

Comment:
At this point I think it's worth investigating (in a different PR) getting this in it's own file. It needs syntax highlighting and to be linted/formatted/etc...

Reply:
https://github.com/pantsbuild/pants/pull/14080